gdal/doc/source/drivers/raster/grass.rst

```diff
@@ -69,15 +69,15 @@ Notes on driver variations
 --------------------------
 
 For GRASS 5.7 Radim Blazek has moved the driver to using the GRASS
-shared libraries directly instead of using libgrass. Currently (GDAL
-1.2.2 and later) both version of the driver are available and can be
-configured using ``--with-libgrass`` for the libgrass variant or
-``--with-grass=<dir>`` for the new GRASS 5.7+ library based version. The
-GRASS 5.7+ driver version is currently not supporting coordinate system
-access, though it is hoped that will be corrected at some point.
+shared libraries directly instead of using libgrass. Currently both
```

Comment:
this needs to be revised

Reply:
I am not sure how to simplify `gdal/configure.ac`.

Reply:
I see @landam was the last one to too the GRASS part in configure.ac. My understanding of it + your changes is that this should still work unmodified against GRASS 7.x. Simplifying configure.ac would be indeed nice, but could be a work for later I guess

Reply:
That's of course fine for me.

Reply:
my point was about just the .rst. The mentions for "new GRASS 5.7+" could be simplified to just mentioning GRASS 7

Reply:
Updated accordingly.